Sofa/Component/Engine/Analyze/src/sofa/component/engine/analyze/Distances.inl

```diff
@@ -47,7 +47,7 @@ Distances< DataTypes >::Distances ( sofa::component::topology::container::dynami
     showGradientMap ( initData ( &showGradientMap, false, "showGradients","show gradients for each point of the target point set." ) ),
     showGradientsScaleFactor ( initData ( &showGradientsScaleFactor, 0.1, "showGradientsScaleFactor","scale for the gradients displayed." ) ),
     offset ( initData ( &offset, Coord(), "offset","translation offset between the topology and the point set." ) ),
-    distanceType ( initData ( &distanceType, TYPE_GEODESIC, "distanceType","type of distance to compute for inserted frames." ) ),
+    distanceType ( initData ( &distanceType, {"Geodesic","Harmonic","Stiffness Diffusion", "Voronoi", "Harmonic with Stiffness"}, "distanceType","type of distance to compute for inserted frames." ) ),
```

Comment:
I don't understand in the code where is the linked done between for example the string "Geodesic" and `#define TYPE_GEODESIC 0` used in the methods.
Did you just fixed the crash or could you test the component?

Reply:
I just fixed the crash. The macro TYPE_GEODESIC corresponds to the position of the string "geodesic" in the list of options.

Reply:
I shouldn't have asked... not sure I can approve this PR anymore 🤣